test_libs/pyspec/eth2spec/test/helpers/deposits.py

```diff
@@ -4,25 +4,31 @@
 from eth2spec.utils.ssz.ssz_impl import signing_root
 
 
-def build_deposit_data(spec, state, pubkey, privkey, amount, withdrawal_credentials, signed=False):
+def build_deposit_data(spec, pubkey, privkey, amount, withdrawal_credentials, state=None, signed=False):
     deposit_data = spec.DepositData(
         pubkey=pubkey,
         withdrawal_credentials=withdrawal_credentials,
         amount=amount,
     )
     if signed:
-        sign_deposit_data(spec, state, deposit_data, privkey)
+        sign_deposit_data(spec, deposit_data, privkey, state)
```

Comment:
Looks unnecessary, just hold the `spec, state, args...` order (as introduced by Carl in phase-1 PR) everywhere for consistency please.

Reply:
because I changed `state` to optional. (`def sign_deposit_data(spec, deposit_data, privkey, state=None)`) It should be `sign_deposit_data(spec, deposit_data, privkey, state=state)`.